<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PinBox</class>
 <widget class="QWidget" name="PinBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>376</width>
    <height>334</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PinBox</string>
  </property>
  <property name="styleSheet">
   <string>background-color: rgb(130, 130, 130);</string>
  </property>
  <widget class="QGroupBox" name="groupPincode">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>371</width>
     <height>152</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>7</y>
      <width>361</width>
      <height>36</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string>background-color: qlineargradient(spread:reflect, x1:0.988636, y1:0, x2:0.989, y2:0.465909, stop:0 rgba(156, 156, 156, 255), stop:1 rgba(194, 194, 194, 255));</string>
    </property>
    <property name="text">
     <string>PIN CODE</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPinExit">
    <property name="geometry">
     <rect>
      <x>321</x>
      <y>14</y>
      <width>26</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/Resources/stop.png</normaloff>:/Resources/stop.png</iconset>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lblPin">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>280</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>36</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string>color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>12345678</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupRequest">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>170</y>
     <width>371</width>
     <height>152</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>7</y>
      <width>361</width>
      <height>36</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string>background-color: qlineargradient(spread:reflect, x1:0.988636, y1:0, x2:0.989, y2:0.465909, stop:0 rgba(156, 156, 156, 255), stop:1 rgba(194, 194, 194, 255));</string>
    </property>
    <property name="text">
     <string>REQUEST  CONNECT</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRequestExit">
    <property name="geometry">
     <rect>
      <x>321</x>
      <y>14</y>
      <width>26</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/Resources/stop.png</normaloff>:/Resources/stop.png</iconset>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lblPinMessge">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>311</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string>color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>12345678</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btnYes">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>120</y>
      <width>51</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string>color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Yes</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnNo">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>120</y>
      <width>51</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string>color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>No</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="TSDClient.qrc"/>
 </resources>
 <connections/>
</ui>
